app/models/front_end_builds/build.rb

```diff
@@ -8,6 +8,7 @@ class Build < ActiveRecord::Base
                       :endpoint
     end
 
+    has_one :live_app, class_name: "FrontEndBuilds::App", foreign_key: :live_build_id
```

Comment:
Also is a trait possible to clean up tests?

```rb
let!(:live_build) do
  FactoryGirl.create :front_end_builds_build, app: app, :live
end

# factories/front_end_builds_build.rb
FactoryGirl.define do
  factory :front_end_builds_build, :class => 'FrontEndBuilds::Build' do
  ...
  association :app, factory: :front_end_builds_app

  trait :live do
    app.live_build = self
  end
```

I tried that but it didn't work.